internal/amt/commands.go

```diff
@@ -37,12 +37,13 @@ type CodeVersions struct {
 
 // InterfaceSettings ...
 type InterfaceSettings struct {
-	IsEnabled   bool   `json:"isEnable"`
-	LinkStatus  string `json:"linkStatus"`
-	DHCPEnabled bool   `json:"dhcpEnabled"`
-	DHCPMode    string `json:"dhcpMode"`
-	IPAddress   string `json:"ipAddress"` //net.IP
-	MACAddress  string `json:"macAddress"`
+	IsEnabled    bool   `json:"isEnable"`
+	LinkStatus   string `json:"linkStatus"`
+	DHCPEnabled  bool   `json:"dhcpEnabled"`
+	DHCPMode     string `json:"dhcpMode"`
+	AmtIPAddress string `json:"amtipAddress"` //net.IP
```

Comment:
I wouldn't recommend changing the previous naming -- this would be considered a breaking change if something else depended on the naming.

```suggestion
	IPAddress string `json:"ipAddress"` //net.IP
```